crates/flox-rust-sdk/src/models/environment_ref.rs

```diff
@@ -29,7 +29,7 @@ impl FromStr for EnvironmentOwner {
 }
 
 #[derive(Debug, Clone, PartialEq, AsRef, Display)]
-pub struct EnvironmentName(String);
+pub struct EnvironmentName(pub String);
```

Comment:
Generally the point of wrappers like this is so that consumers can't rely on the internals. I've personally seen how much of a hassle it refactoring a codebase that went from `pub struct Foo(pub bar)` to something like `pub struct Foo { bar: (), baz: ()}`. It's probably better to implement a conversion trait instead like `Display` or `From<EnvironmentName> for String` so that consumers don't have to rely on the internal structure of `EnvironmentName`.